spec/rubocop/cop/lint/unused_block_argument_spec.rb

```diff
@@ -143,6 +144,31 @@
       end
     end
 
+    context 'when an optional keyword argument is unused', ruby: 2 do
+      let(:source) { <<-END }
+        define_method(:foo) do |bar: 'default'|
+          puts 'bar'
+        end
+      END
+
+      it 'registers an offense but does not suggest underscore-prefix' do
+        expect(cop.offenses.size).to eq(1)
+        expect(cop.highlights).to eq(['bar'])
+        expect(cop.offenses.first.message).to eq(
+          'Unused block argument - `bar`. ' \
+          "You can omit the argument if you don't care about it."
```

Comment:
Well, this message didn't really take keyword args into account. Maybe it should be different for them?